load/scan.go

```diff
@@ -24,27 +24,28 @@ func ackAndMaybeSend(ch *duplexChannel, count *int, unsent []Batch) []Batch {
 // sendOrQueueBatch attempts to send a Batch of data on a duplexChannel; if it would block
 // or there is other work to be sent first, the Batch is stored on a queue. The count of
 // outstanding work is adjusted upwards
-func sendOrQueueBatch(ch *duplexChannel, count *int, batch Batch, unsent []Batch) []Batch {
+func sendOrQueueBatch(ch *duplexChannel, count *int, batch Batch, outstandingBatches []Batch) []Batch {
```

Comment:
don't think this parameter name change is improvement.

In general we try to keep the names somewhat short

Reply:
Ok, agreed.
Summary: updated